tests/python/unittest/test_numpy_op.py

```diff
@@ -240,27 +236,38 @@ def is_int(dtype):
     in_data_dim = random.choice([2, 3, 4])
     shape = rand_shape_nd(in_data_dim, dim=3)
     acc_type = {'float16': 'float32', 'float32': 'float64', 'float64': 'float64',
-                'int8': 'int32', 'int32': 'int64', 'int64': 'int64'}
+                'int8': 'int32', 'int32': 'int64', 'int64': 'int64', 'bool': 'int64'}
     for hybridize in [False, True]:
```

Comment:
would using https://docs.python.org/3.7/library/itertools.html#itertools.product help readability of the code and make it less nested?

Reply:
Thanks for the suggestion. Will consider refactoring it in the following PRs.